server/test/models/deployment_test.rb

```diff
@@ -11,9 +11,29 @@ class DeploymentTest < ActiveSupport::TestCase
       Net::DNS::ARecord.any_instance.stubs(:conflicts).returns([])
     end
 
+    test "should save deployment with valid fields for rhv and cfme" do
+      deployment = fusor_deployments(:rhev_and_cfme)
+      assert deployment.save, "unable to save valid deployment with rhv and cfme"
+    end
+
```

Comment:
Added more "positive" tests so the we're checking valid deployments are saving appropriately.